operations/service_deploy_smoke_utils.go

```diff
@@ -20,7 +21,8 @@ const (
 	// uiPort is the local port the UI will listen on.
 	smokeUiPort = ":9090"
 	// urlPrefix is the localhost prefix for accessing local Evergreen.
-	smokeUrlPrefix = "http://localhost"
+	smokeUrlPrefix  = "http://localhost"
+	containerTaskId = "evergreen_container_bv_container_task_a71e20e60918bb97d41422e94d04822be2a22e8e_22_08_22_13_44_49"
```

Comment:
nit: can we call this `smokeContainerTaskID` or something like that?

Reply:
Sure, done